### PR TITLE
Add info about resolver arguments

### PR DESCRIPTION
**Pull request by atwixfirster**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will add information about resolvers parameters on  "Resolvers" page.

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/graphql/develop/resolvers.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/develop/resolvers.md [hunk 1]
```diff
@@ -5,6 +5,42 @@ title: Resolvers
 
 A resolver performs GraphQL request processing. In general, it is responsible for constructing a query, fetching data and performing any calculations, then transforming the fetched and calculated data into a GraphQL array format. Finally, it returns the results wrapped by a callable function.
 
+There are the following required arguments for resolver:
```
Comment (keharper) [inline guides/v2.3/graphql/develop/resolvers.md]:
```suggestion
A resolver requires the following arguments:
```

File: guides/v2.3/graphql/develop/resolvers.md [hunk 2]
```diff
@@ -5,6 +5,42 @@ title: Resolvers
...
+        array $args = null
+    );
+```
+
+Field | Type | Description
+--- | --- | ---
+$field | [`Magento\Framework\GraphQl\Config\Element\Field`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Config/Element/Field.php) | Fields are used to describe possible values for a type/interface
+$context | [`Magento\Framework\GraphQl\Query\Resolver\ContextInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/Resolver/ContextInterface.php) | Resolver Context is used as a shared data extensible object in all resolvers that implement [`ResolverInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php).
+$info | [`Magento\Framework\GraphQl\Schema\Type\ResolveInfo`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Schema/Type/ResolveInfo.php) | Structure containing information useful for field resolution process.
+$value | array | Contains additional query parameters. `Null` in the most of cases.
```
Comment (keharper) [inline guides/v2.3/graphql/develop/resolvers.md]:
```suggestion
$value | array | Contains additional query parameters. `Null` in most cases. 
```

File: guides/v2.3/graphql/develop/resolvers.md [hunk 3]
```diff
@@ -5,6 +5,42 @@ title: Resolvers
...
+        ResolveInfo $info,
+        array $value = null,
+        array $args = null
+    );
+```
+
+Field | Type | Description
+--- | --- | ---
+$field | [`Magento\Framework\GraphQl\Config\Element\Field`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Config/Element/Field.php) | Fields are used to describe possible values for a type/interface
+$context | [`Magento\Framework\GraphQl\Query\Resolver\ContextInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/Resolver/ContextInterface.php) | Resolver Context is used as a shared data extensible object in all resolvers that implement [`ResolverInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php).
```
Comment (keharper) [inline guides/v2.3/graphql/develop/resolvers.md]:
```suggestion
$context | [`Magento\Framework\GraphQl\Query\Resolver\ContextInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/Resolver/ContextInterface.php) | Resolver context is used as a shared data extensible object in all resolvers that implement [`ResolverInterface`]({{ site.mage2300url }}lib/internal/Magento/Framework/GraphQl/Query/ResolverInterface.php).
```

Comment (atwixfirster) [reply guides/v2.3/graphql/develop/resolvers.md]: **fixed**

File: guides/v2.3/graphql/develop/resolvers.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/develop/resolvers.md]: **fixed**

File: guides/v2.3/graphql/develop/resolvers.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/develop/resolvers.md]: **fixed**

**Comment by atwixfirster** [comment]
> A few small changes, @atwixfirster

**implemented**

Thank you, @keharper !

**Comment by keharper** [comment]
Running tests

**Pull request merged**